bzip2-sys/lib.rs

```diff
@@ -2,6 +2,13 @@
 
 extern crate libc;
 
+#[cfg(target_arch = "wasm32")]
+mod wasm_shim;
```

Comment:
Could this be behind something like `cfg(wasm_shim)` which is printed out by the build script?